test/EndToEnd/tests/ExecuteInitScriptTest.ps1

```diff
@@ -0,0 +1,150 @@
+function Test-NetCoreProjectExecuteInitScriptOnInstall {
```

Comment:
Does the CI process pick up new E2E test scripts like this through a wildcard reference (e.g.:  run all *.ps1 in this directory)?